.github/workflows/apt.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           - Ubuntu Focal amd64
           - Ubuntu Jammy amd64
           - Ubuntu Noble amd64
+        upgrade_to: [5, 6]
```

Comment:
Thanks for this fix!
I haven't seen the changes in detail yet, but I have a question why we need this.

I feel that mixing it with existing tests would make things complicated.
There are already `update-to-next-version` tests.
If we need to test something like `update-to-next-major-version` tests, shouldn't it be the one test file?

Looks like the `install-missing-plugins` test needs this.
Is there any difficulty, for example, in doing that test within the existing `update-to-next-version` test?

Reply:
> Looks like the `install-missing-plugins` test needs this.
> Is there any difficulty, for example, in doing that test within the existing `update-to-next-version` test?

Or maybe we should add `update-to-next-major-version` test file.

Reply:
Adding update-to-next-major-version  might be better.
I'll fix it.